clicommand/pipeline_upload.go

```diff
@@ -320,7 +320,7 @@ var PipelineUploadCommand = cli.Command{
 
 			return err
 			// On a server error, it means there is downtime or other problems, we
-			// need to retry. Let's retry every 5 seconds, for a total of 5 minutes.
+			// need to roko. Let's retry every 5 seconds, for a total of 5 minutes.
```

Comment:
Ah.... Maybe not this one, as good as it sounds 😅

Reply:
🙈 omg